seednsort.md

```diff
@@ -23,4 +23,4 @@ The Seeders/Sorters team will use the EDGI subprimer systems ([found here](https
   - In the spreadsheet is automatically associated with a universal unique identifyer (UUID) that was generated in advance.
 
```

Comment:
Can we add in the Wayback Chrome extension? It is a nice spot check to see how things are saved in IA